### ISSUE TITLE
Please add tags and releases of Tag Manager

### ISSUE THREAD

**Issue by mikkeschiren** [matomo-org/tag-manager#940]
It is hard to follow changes in this plugin, and as commits are added to the core release of Matomo. There should be tags and releases, this is not following semver.

**Comment by AltamashShaikh** [comment]
@mikkeschiren Will see if we can do anything better here, just FYI we do not follow semver in case of TagManager as it is coupled with Matomo core and hence there is no release for this plugin separately, unlike other Matomo plugins.

**Comment by mikkeschiren** [comment]
Yeah, do understand that - but some of the plugins that is included in core has releases, and some do not. It would make sense if all has, and in the bigger picture (noothing to do with Tag Manager per see) - that tagged releases are in core release. It would send a signal of maturity nevertheless.

**Comment by sgiehl** [comment]
@mikkeschiren Actually no plugin included in core releases have their own version numbers.
The github repo has a couple of plugins included as submodules, which have their own releases. But those are not included in a release. The only exception is the TagManager.

**Comment by mikkeschiren** [comment]
My point here where about the sub models that are in a release of Matomo, sorry for the confusion :)

**Comment by Stan-vw** [comment]
Good ticket. We've recently implemented a process internally that should mean all MTM release notes will now be part of the core release notes. 
With that, I'll close the ticket, but feel free to reopen in case the changes don't seem to be properly reflected in release notes at any point 👍
